### ISSUE TITLE
Why Train loss anf Test Loss nan for all the epochs?

### ISSUE THREAD

**Issue by Pari-singh** [YunYang1994/tensorflow-yolov3#273]
Hi, I followed the instruction given on readme and made the same changes in config file. I also got the dataset csv file in the same format as is mentioned, having the location of the image with its name inside the csv file. But still when I run the training, I am constantly getting nan for both Train and test loss. If the model is not able to read something, it should throw error rather than just running something unnecessary.

Can someone help?

**Comment by xiao12mm** [comment]
https://github.com/YunYang1994/tensorflow-yolov3/issues/294
